src/quo/components/counter/counter/style.cljs

```diff
@@ -3,31 +3,35 @@
     [quo.foundations.colors :as colors]))
 
 (defn get-color
-  [type customization-color theme]
+  [type customization-color theme blur?]
   (case type
     :default   (colors/resolve-color customization-color theme)
     :secondary (colors/theme-colors colors/neutral-80-opa-5 colors/white-opa-5 theme)
-    :grey      (colors/theme-colors colors/neutral-10 colors/neutral-80 theme)
+    :grey      (if blur?
+                 (colors/theme-colors colors/neutral-80-opa-10 colors/white-opa-10 theme)
```

Comment:
Implements only the required variant for the checkboxes, this component is incomplete, it's been reported here:
- https://github.com/status-im/status-mobile/issues/22386